### PR TITLE
Set export on Config to verbose

### PR DESCRIPTION
**Pull request by renatoalmeidaoliveira**
# SUMMARY
In the configuration export session i think that using /export verbose is better than just /export, because with export only some configurations sections are going to be missed like BGP configuration, as proposed in https://github.com/ansible/ansible/pull/67868

# ISSUE TYPE
Minor Change Request
COMPONENT NAME
routeros_facts.py

### extract of the output with /export verbose

```

# software id = 
#
#
#
/interface ethernet
set [ find default-name=ether1 ] advertise=\
    10M-half,10M-full,100M-half,100M-full,1000M-full arp=enabled arp-timeout=\
    auto auto-negotiation=yes cable-settings=default disable-running-check=\
    yes disabled=no full-duplex=yes loop-protect=default \
    loop-protect-disable-time=5m loop-protect-send-interval=5s mac-address=\
    50:00:00:02:00:00 mtu=1500 name=ether1 orig-mac-address=50:00:00:02:00:00 \
    rx-flow-control=off speed=1Gbps tx-flow-control=off
set [ find default-name=ether2 ] advertise=\
    10M-half,10M-full,100M-half,100M-full,1000M-full arp=enabled arp-timeout=\
    auto auto-negotiation=yes cable-settings=default disable-running-check=\
    yes disabled=no full-duplex=yes loop-protect=default \
    loop-protect-disable-time=5m loop-protect-send-interval=5s mac-address=\
    50:00:00:02:00:01 mtu=1500 name=ether2 orig-mac-address=50:00:00:02:00:01 \
    rx-flow-control=off speed=1Gbps tx-flow-control=off
set [ find default-name=ether3 ] advertise=\ 
.....
.....
```
### Extract of Output with only /export:
```
# software id = 
#
#
#
/interface wireless security-profiles
set [ find default=yes ] supplicant-identity=MikroTik
/ip dhcp-client
add dhcp-options=hostname,clientid disabled=no interface=ether1
<END>
```

### CODE REVIEW

**Comment by felixfontein** [comment]
Could you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to) (probably `minor_changes`)?

CC @heuels

File: changelogs/fragments/104_routeros_facts_export_verbose [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+ - set configuration export on routeros_facts to verbose, for full configuration export
```
Comment (felixfontein) [inline changelogs/fragments/104_routeros_facts_export_verbose:2]:
```suggestion
 - routeros_facts - set configuration export on to verbose, for full configuration export (https://github.com/ansible-collections/community.network/pull/104).
```

**Comment by felixfontein** [comment]
You probably also need to rename the unit test fixture file (as you did in ansible/ansible#67868). It now lives in tests/unit/plugins/modules/network/routeros/fixtures/routeros_facts/.

**Comment by felixfontein** [comment]
Looks good to me. @heuels what do you think?

**Comment by heuels** [comment]
@felixfontein, yes, it looks great – let's merge it 🙂

**Comment by felixfontein** [comment]
@heuels aaaand... done! :) thanks for reviewing this!
@renatoalmeidaoliveira thanks a lot for implementing this!

**Pull request merged**